src/execution/vm/vm.cpp

```diff
@@ -2112,9 +2112,9 @@ void VM::Interpret(const uint8_t *ip, Frame *frame) {  // NOLINT
   OP(Concat) : {
     auto *result = frame->LocalAt<sql::StringVal *>(READ_LOCAL_ID());
     auto *exec_ctx = frame->LocalAt<exec::ExecutionContext *>(READ_LOCAL_ID());
-    auto *left = frame->LocalAt<const sql::StringVal *>(READ_LOCAL_ID());
-    auto *right = frame->LocalAt<const sql::StringVal *>(READ_LOCAL_ID());
-    OpConcat(result, exec_ctx, left, right);
+    auto **inputs = frame->LocalAt<const sql::StringVal **>(READ_LOCAL_ID());
+    auto num_inputs = READ_UIMM4();
+    OpConcat(result, exec_ctx, num_inputs, inputs);
```

Comment:
Nit: try to be consistent with parameter ordering between bytecode and C++ functions.

Reply:
Updated